service/src/main/java/com/epam/rft/atsy/service/impl/CandidateServiceImpl.java

```diff
@@ -183,10 +182,18 @@ private void validateCandidateFilterRequest(CandidateFilterRequest candidateFilt
 
       try {
         Sort.Direction.fromString(candidateFilterRequest.getSortOrder());
-      } catch (IllegalArgumentException e) {
+      } catch (Exception e) {
         throw new IllegalArgumentException(
             "Invalid sort order: " + candidateFilterRequest.getSortOrder(), e);
       }
+
+      try {
```

Comment:
I do not really understand the reason of this try block. Is it necessary? A new PageRequest object is created but nothing to do with it.